### ISSUE TITLE
 Fine tuning of UCF101 and HMDB51

### ISSUE THREAD

**Issue by LV0913** [MCG-NJU/TDN#28]
If I want to do fine tuning training on UCF101 and HMDB51, how should I prepare the dataset and train it?

**Comment by yztongzhan** [comment]
You can refer [here](https://github.com/MCG-NJU/TDN/issues/8)

**Comment by LV0913** [comment]
Thanks! But my dataset contains only videos. How can I get the annotation files required in the code?
like `        filename_imglist_train = "hmdb51_rgb_train_split_3.txt"
        filename_imglist_val = "hmdb51_rgb_val_split_3.txt"`
or `        filename_imglist_train = "ucfTrainTestlist/ucf101_rgb_train_split_3.txt"
        filename_imglist_val = "ucfTrainTestlist/ucf101_rgb_val_split_3.txt"`

**Comment by yztongzhan** [comment]
Please refer the [TSN](https://github.com/yjxiong/tsn-pytorch) and [TSM](https://github.com/mit-han-lab/temporal-shift-module) codebase for more details.

**Comment by sean186** [comment]
Hi @yztongzhan ，I have trained ucf101，got best prec@1:85.0 (paper 97.4%)
```
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=3 \
    main.py ucf101 RGB --arch resnet50  --num_segments 16 --gd 20 --lr 0.001 \
    --lr_steps 10 20 --epochs 25 --batch-size 8 --dropout 0.8 --consensus_type=avg \
    --eval-freq=1 -j 8  --npb 
```
Use this command, I did not get the approximate result of the paper. Can you share how you did it, Thank you.
Can you share how to use kinetics pretrain?  Thank you very much

**Comment by yztongzhan** [comment]
Hi @sean186! To reproduce our results on UCF101, you need to prepare the pre-trained model and add this argument `--tune_from='checkpoint/best.pth.tar` to your command. We use the following script for fine-tuning:
```
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=8 \
main.py hmdb51 RGB --arch resnet50  --num_segments 16 --gd 20 --lr 0.002 \
--lr_steps 10 20 --epochs 25 --batch-size 16 --dropout 0.8 --consensus_type=avg \
--eval-freq=1 -j 4 --tune_from='checkpoint/best.pth.tar' --npb 
```
Then we perform 10-clip and 3-crop for accuracy evaluation protocol following [TEA](https://arxiv.org/abs/2004.01398) and [STM](https://arxiv.org/abs/1908.02486).
Considering that you only use 3 GPUs (because you set `--nproc_per_node=3`) for your experiment, we recommend that you modify your learning rate according to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677)
